arithmetization/src/main/java/net/consensys/linea/zktracer/container/stacked/StackedSet.java

```diff
@@ -56,9 +60,17 @@ public void pop() {
   }
 
   public boolean add(E e) {
-    if (!operationsCommitedToTheConflation().contains(e)) {
-      return operationsInTransaction().add(e);
+    return operationsInTransaction().add(e);
```

Comment:
This is the real change : we just check if it's a new operation prior to this transaction only